src/like-service/app/Http/Controllers/DoLikeController.php

```diff
@@ -4,10 +4,39 @@
 use Illuminate\Foundation\Bus\DispatchesJobs;
 use Illuminate\Foundation\Validation\ValidatesRequests;
 use Illuminate\Routing\Controller as BaseController;
-public class DoLikeController extends BaseController{
+use Illuminate\Http\Request;
```

Comment:
[ISSUE] Please remove unused import

Please in general reformat this file. There is a bunch of extra spaces. The parenthesis and indentation is also not very "php like". After running the auto-formatter, it looks like this for me:

```
<?php

use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
use Illuminate\Foundation\Bus\DispatchesJobs;
use Illuminate\Foundation\Validation\ValidatesRequests;
use Illuminate\Routing\Controller as BaseController;
use Illuminate\Support\Facades\Http;
use Illuminate\Support\Facades\Log;

class DoLikeController extends BaseController
{
    use AuthorizesRequests, DispatchesJobs, ValidatesRequests;

    function doLike($request)
    {
        $user_token = $request->cookie('jwt');

        if (!$this->validateToken($user_token)) {
            return response()->json([
                'message' => 'Unauthorized'
            ], 401);
        } else {
            //Todo: do stuff
            Log::notice('Do stuff');
        }
    }

    function validateToken($user_token)
    {

        $auth_service_validate_host = config('app.auth_service_url');

        Log::notice('Sending request to: ' . $auth_service_validate_host);

        $response = Http::post($auth_service_validate_host, [
            'jwt' => $user_token
        ]);

        $responseCode = $response->getStatusCode();
        return $responseCode == 200;
    }
}
```

Reply:
Removed and reformatted using IntelliJ auto format